apps/api/src/app/subscribers-v2/subscribers.module.ts

```diff
@@ -73,6 +74,7 @@ const DAL_MODELS = [
 ];
 
 @Module({
+  imports: [TopicsV2Module],
```

Comment:
Is there a reason to import TopicsV2Module here?
from quick look looks like we need ListSubscriberSubscriptionsUsecase in Subscriber Module, while it lives in TopicModule and it is not used inside of it.
perhaps we need to relocate ListSubscriberSubscriptionsUsecase to SubscriberModule and remove this import?